scalyr_agent/tests/configuration_test.py

```diff
@@ -1557,6 +1558,97 @@ def test_empty_substitution(self):
 
         self.assertEquals(config.api_key, "hi")
 
+    def test_print_config_doesnt_throw(self):
```

Comment:
I would just call this `test_print_config`.  It's inherent in all tests that we don't want the method to unexpectantly throw an exception.

Reply:
Done.